lib/Bio/EnsEMBL/DataCheck/Checks/ExonBounds.pm

```diff
@@ -117,11 +152,11 @@ sub tests {
 
     if (defined $last_transcript_id && $last_transcript_id == $transcript_id) {
       if ($strand == 1) {
-        if ($last_end > $start) {
+        if ($last_end > $start && $last_start < $start) {
```

Comment:
Can I suggest rewriting this check using only `<` so the condition `last_start < start < last_end` is clearer
```suggestion
        if ($last_start < $start && $start < $last_end) {
```